.env-dist

```diff
@@ -139,7 +139,7 @@ SUBSCRIPTION_BILLING_AMOUNT_MONTHLY_US=42.42
 # was released to the welcome flow. Users who had signed up before and thus
 # have seen data breach results before, will be able to see their known breaches
 # first:
-BROKER_SCAN_RELEASE_DATE=2024-02-06
+BROKER_SCAN_RELEASE_DATE=2025-02-06
```

Comment:
This will make it the default for our actual deployments as well. The variable should be set there, but it's still riskier than I would like. Could we set this in just for the E2E test runs? i.e. like [here](https://github.com/mozilla/blurts-server/pull/4169/files#diff-6175e3066c9d2ba5a9bd6846a8aaf039cbe9ba5554558e06ac273bc295a8c627R82-R84) (and maybe also in `e2e_cron.yml`? Though that might need more work, and the tests should be updated to not need this variable change anyway.).

Reply:
In that case, that’s too risky indeed. Updated to apply the same change you linked in [2f2344c](https://github.com/mozilla/blurts-server/pull/4174/commits/2f2344c7c1a4a8cf8e03fd813bf064cd6b64a927)